src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -682,4 +688,4 @@ static uct_mm_md_mapper_ops_t uct_posix_md_ops = {
 };
 
 UCT_MM_TL_DEFINE(posix, &uct_posix_md_ops, uct_posix_rkey_unpack,
-                 uct_posix_rkey_release, "POSIX_")
+                 uct_posix_rkey_release, "POSIX_", uct_posix_iface_config_table)
```

Comment:
can we use uct_mm_iface_config_table?

Reply:
in this case we will not have common config vars suitable for all MM transports. 
Also `UCX_MM_...` will not be accepted by UCX anymore, which breaks existing env vars naming

Reply:
i do not understand why passing uct_mm_iface_config_table instead of uct_posix_iface_config_table will make MM_ not work..

Reply:
it would remove common parent table

Reply:
ok